tests/shared/src/test/scala/scala/meta/tests/tokenizers/TokenizerSuite.scala

```diff
@@ -1089,4 +1089,9 @@ class TokenizerSuite extends BaseTokenizerSuite {
       ("raw\"\"\"\\$host\\$share\\\"\"\"").tokenize.get
 
   }
+
+  test("wrapped-identifiers-ApplyInfix") {
```

Comment:
Shouldn't this be in the TermSuite? Maybe not really needed with the BasicPositionSuite already testing it?

Reply:
https://github.com/scalameta/scalameta/pull/2736/commits/f178af78f449d9fd12b8a5359e8afc05c2f4b20c